### ISSUE TITLE
When building container, error in terminal

### ISSUE THREAD

**Issue by anthonywebb** [electrode-io/electrode-native#193]
Hi Guys,  a colleague of mine and I are trying to get an end to end hello world example running where er have a basic miniapp in a cauldron and publishing a container to git.

After downloading the container, if I open the xcodeproj andtry to build it, a terminal pops up and gives an error:
```
module.js:471
    throw err;
    ^

Error: Cannot find module '/Users/antwebb/Downloads/assistant_service-container-master/ElectrodeContainer/Libraries/ReactNative/local-cli/cli.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
Process terminated. Press <enter> to close the window
```

The build does appear to compile fine, but we are having some issues including it in a fresh host app so we are trying to back into why:
<img width="980" alt="screen shot 2017-10-13 at 8 45 54 am" src="https://user-images.githubusercontent.com/4397/31552666-628b1de8-aff5-11e7-85c8-c08dd33b259f.png">

**Comment by anthonywebb** [comment]
I should note that the above error happens when the container is built using the beta version of xcode (9.1) I downloaded the latest stable Version 9.0 (9A235), and pushed an update to generate a new container.  Now when I download and open that container and try a build it will not succeed like it was previously.  Here is what I see when attempting the build on the container:

<img width="1116" alt="screen shot 2017-10-13 at 10 59 17 am" src="https://user-images.githubusercontent.com/4397/31557511-48831368-b006-11e7-87db-f63276735e33.png">

**Comment by clairewli** [comment]
hey @anthonywebb, with respect to the `error` you see in terminal, that's some noise from react-native, we are working on cleaning it up. As long as your project compile file, you don't need to worry about it at the moment. 

In your second screenshot, notice that you are trying to build `ElectrodeContainer` to a `Generic iOS Device`. Can you try to build it to a specific simulator like `iphone 8 plus` or `iphone x`?

**Comment by anthonywebb** [comment]
Hi @clairewli thanks for your help..  When I select iPhone8Plus I get:

<img width="1114" alt="screen shot 2017-10-13 at 11 25 50 am" src="https://user-images.githubusercontent.com/4397/31558273-7cbac83a-b009-11e7-903d-36b7c626d23b.png">

**Comment by clairewli** [comment]
hey @anthonywebb, do you happen to have your container in your public repo? I didn't see the `iterator_adator.hpp` file on my local machine.

**Comment by anthonywebb** [comment]
@clairewli my container is on a private repo on our VPN. I am reading up on that file here, wondering if we are having similar issues with the container generator? https://github.com/facebook/react-native/issues/14423

**Comment by anthonywebb** [comment]
@clairewli good news!  I have been able to recreate the container, clone it, and it is compiling fine now!  I suspect that this might be cause due to the fact that the original container I created was with only the xocde beta installed (9.1) but I later rolled back to 9.0 and was creating more containers.  I suspect that perhaps this was the issue?  At any rate, fresh container generated and is compiling fine.  So I am off to figure out why all these headers are missing? https://github.com/electrode-io/electrode-native/issues/195
